Example/Example/Assets.xcassets/play-button.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "play-button.png",
```

Comment:
Set `single scale` please, and ideally change to vector PDF button? Also, make sure this "black" button works in dark mode, if not, we will need light and dark version of the asset :)